services.json

```diff
@@ -1,11 +1,12 @@
 {
   "services": {
     "pub": {
-      "publishes": ["load_test", "pricing_changed"],
+      "publishes": ["load_test", "pricing_changed", "bulked_msg"],
       "concurrency": 100
     },
     "sub": {
-      "subscribes": ["load_test", "pricing_changed"],
+      "maintenance": true,
+      "subscribes": ["load_test", "pricing_changed", "bulked_msg@3"],
```

Comment:
It'll be much more clear if it was an object with a concurrency property. easier to read and less chances for confusion. also, with this it's impossible to use `@` in the event name.